pkgs/development/python-modules/ffcv/default.nix

```diff
@@ -10,13 +11,21 @@
   pytorch-pfn-extras,
   terminaltables,
   tqdm,
+  pytestCheckHook,
+  assertpy,
+  psutil,
+  torchvision,
+  webdataset,
 }:
 
 buildPythonPackage rec {
   pname = "ffcv";
   version = "1.0.0";
   format = "setuptools";
```

Comment:
Upkeep/refactoring/not mandatory: `format = "pyproject"` and `build-system = [ setuptools ]` are the preferred alternative.

Reply:
Actually I think the new method is `pyproject = true/false` based on this: https://github.com/NixOS/nixpkgs/issues/253154#issuecomment-1712673789

Reply:
Oh, yes, you're right, my thinking just hadn't caught up with the typing...